docs/topics/testing/advanced.txt

```diff
@@ -421,19 +421,79 @@ following structure:
 
 .. code-block:: text
 
-    runtests.py
     polls/
         __init__.py
         models.py
         ...
     tests/
         __init__.py
         models.py
-        test_settings.py
+        settings.py
         tests.py
+        urls.py
 
 Let's take a look inside a couple of those files:
 
+.. code-block:: python
+   :caption: ``tests/settings.py``
+
+   from django.urls import include, path
+
+   SECRET_KEY = "fake-key"
+
+   INSTALLED_APPS = [
+       "polls",
+       # adding tests allows us to define models inside the tests package
+       # "tests",
```

Comment:
Should `tests` be commented out?

Reply:
No, that probably doesn't add any value. I added it for the introductory case where it serves as a pointer but no test models are actually present in the intro app.